.github/workflows/trigger_release.yml

```diff
@@ -0,0 +1,46 @@
+name: Trigger Release
+
+# This workflow triggers a release creation with changelog and the release notes created by the release toolkit.
+# This workflow should be triggered merely from the master branch.
+# If you wish to be 100% free creating a pre-release, just create it manually.
+
+on:
+  workflow_dispatch:
+## TODO here we will need a cronjob.
+
+jobs:
+  release:
+    name: Create release
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.version.outputs.next-version }}
+      image-tag: ${{ steps.image-tag.outputs.tag }}
+    steps:
+      - name: Check is triggered for master branch
+        run: |
+          if [ ${{ env.GITHUB_REF_NAME }} != "master" ] || [ ${{ env.GITHUB_REF_TYPE }} != "branch" ];then
+            echo This workflow should only be triggered for the 'master' branch
+            exit 1
+          fi
```

Comment:
Nit: I wonder if we could use a reference to the default branch instead of the _harcoded_ 'master' value.

Reply:
I'll look into it!

Reply:
`${{ github.event.repository.default_branch }}`

Should do the job